packages/cashier/src/components/side-note/side-note.tsx

```diff
@@ -18,7 +18,7 @@ type TSideNoteBullet = {
 };
 
 type TSideNoteProps = {
-    children?: TReactChildren[] | TReactElement;
+    children?: JSX.Element | JSX.Element[] | TReactChildren[];
```

Comment:
```suggestion
type TSideNoteProps = React.PropsWithChildren<{
```
@hamid-deriv Can’t we do this instead? 🤔

Reply:
Sorted.